org.civicrm.reqangular/test/models/option-group.spec.js

```diff
@@ -35,12 +35,56 @@ define([
     });
 
     describe('valuesOf()', function () {
-      beforeEach(function () {
-        OptionGroup.valuesOf('hrjc_department');
+      var optionGroupName = 'hrjc_department';
+
+      describe('when just a single option group name is passed', function () {
+        beforeEach(function () {
+          OptionGroup.valuesOf(optionGroupName);
+        });
+
+        it('calls the correct API method', function () {
+          expect(optionGroupAPI.valuesOf.calls.mostRecent().args[0]).toBe(optionGroupName);
+        });
+
+        it('caches requests by default', function () {
```

Comment:
have a 
```js
describe('caching', function () {
});
```
block where you test both the default caching behaviour and the usage of the `cache` param